src/server/index.jsx

```diff
@@ -75,10 +90,14 @@ server
       res.json(articleJSON);
       return null;
     });
-  })
-  .get('/status', (req, res) => {
-    res.sendStatus(200);
-  })
+  });
+}
+
+/*
+ * Test and Live env routes
```

Comment:
```suggestion
 * Application env routes
```

When skimming this file, this comment kinda implies that `local` won't have these routes which is an incorrect assumption. It's difficult to name these routes because they are used _for_ `test` and `live`, but also _exist_ on both `dev` and `prod` 🤔